app/models/derivative.rb

```diff
@@ -53,6 +55,31 @@ def initialize(*args)
     self.managed = true
   end
 
+  alias_method :_width=, :width=
```

Comment:
I thought about making a concern out of it, but because one uses `#original_frame_size` and the other uses `#resolution`, I wasn't sure it was a good idea. I'll take a stab at it though.